dub.json

```diff
@@ -21,17 +25,33 @@
         },
     },
 
+    "subPackages": [
+        {
+            "name": "configure",
+            "targetType": "executable",
```

Comment:
Oh, I have missed that. I was `rdmd` script before, wasn't it? Why have you changed that? Alternatively you can move the package definition to `configure.d` itself:

```
#!/usr/bin/env dub
/+ dub.json:
{
    "name": "configure",
    ...
+/

(D code follows)
```